### PR TITLE
Updates documentation

### LINKED ISSUE #8: Update documentation to the latest

### PR DESCRIPTION
**Pull request by nitinblotout**
Resolves #8

### CODE REVIEW

File: docs/defined_events.md [hunk 1]
```diff
@@ -29,6 +29,6 @@ var data = MapIDData()
 data.externalID = "92j2jr230r-232j9j2342j3-jiji"
 data.provider = "sass"
 
-BlotoutAnalytics.getInstance().mapId(data, null);
-BlotoutAnalytics.getInstance().mapId(data, eventInfo);
+BlotoutAnalytics.INSTANCE..mapID(data, null);
+BlotoutAnalytics.INSTANCE..mapID(data, eventInfo);
```
Comment (NejcZdovc) [inline docs/defined_events.md:33]: `INSTANCE..mapID` -> `INSTANCE.mapID`

Comment (nitinblotout) [reply docs/defined_events.md]: Done.

File: docs/integration.md [hunk 2]
```diff
@@ -1,73 +1,47 @@
...
-implementation fileTree(dir: 'libs', include: ['*.jar','*.aar'])
-implementation ('com.birbit:android-priority-jobqueue:2.0.1')
-implementation ('com.squareup.retrofit2:retrofit:2.6.2')
-implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.8'
-implementation 'com.squareup.retrofit2:converter-gson:2.3.0'
-implementation group: 'com.squareup.retrofit2', name: 'converter-jackson', version: '2.6.2'
-implementation 'com.squareup.okhttp3:logging-interceptor:3.10.0'
-implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.50"
-implementation 'com.android.installreferrer:installreferrer:2.1'
+```Kotlin
```
Comment (NejcZdovc) [inline docs/integration.md:12]: `Kotlin` -> `kotlin`

**Review by NejcZdovc — COMMENTED** [review]
squash commits in one

**Comment by nitinblotout** [comment]
> squash commits in one

Done.

**Review by NejcZdovc — APPROVED** [review]

**Pull request merged**